content/docs/reference/serializing-functions.md

```diff
@@ -126,9 +125,9 @@ const lambda = new aws.lambda.CallbackFunction("mylambda", {
 });
 ```
 
-Then this would need to serialize the entire object value (because 'bar' itself is used from 'foo'). This process happens conservatively. By default objects are serialized in their entirety, and they are only trimmed down when it can be proved that it is totally safe to do so.
+Then Pulumi would need to serialize the entire object value, since `bar` itself is used from `foo`. This process happens conservatively. By default objects are serialized in their entirety, and they are only trimmed down when it can be proved that it is totally safe to do so.
```

Comment:
maybe "can be proven" instead of "can be proved"?  not sure.

Reply:
Oh I missed this one - thank you!